kkm/kkm_guest_entry.S

```diff
@@ -224,7 +224,7 @@ kkm_switch_to_gp_asm:
 	pushq	OFF_RAX(%rdi)	# user %rax
```

Comment:
Is the change in indentation here significant?

Reply:
I don't see any problem with this.
the only rule is one instruction per line and white space it could be anything.